test/Microsoft.ML.Tests/Transformers/FeatureSelectionTests.cs

```diff
@@ -21,30 +24,30 @@ public FeatureSelectionTests(ITestOutputHelper helper)
         {
         }
 
-        [Fact(Skip = "FeatureSeclection transform cannot be trained on empty data, schema propagation fails")]
+        [Fact]
         public void FeatureSelectionWorkout()
         {
             string sentimentDataPath = GetDataPath("wikipedia-detox-250-line-data.tsv");
-            var data = TextLoader.CreateReader(Env, ctx => (
+            var data = TextLoader.CreateReader(ML, ctx => (
                     label: ctx.LoadBool(0),
                     text: ctx.LoadText(1)), hasHeader: true)
                 .Read(sentimentDataPath);
 
-            var invalidData = TextLoader.CreateReader(Env, ctx => (
+            var invalidData = TextLoader.CreateReader(ML, ctx => (
                     label: ctx.LoadBool(0),
                     text: ctx.LoadFloat(1)), hasHeader: true)
                 .Read(sentimentDataPath);
 
-            var est = new WordBagEstimator(Env, "text", "bag_of_words")
-                .Append(new CountFeatureSelector(Env, "bag_of_words", "bag_of_words_count", 10)
-                .Append(new MutualInformationFeatureSelector(Env, "bag_of_words", "bag_of_words_mi", labelColumn: "label")));
+            var est = new WordBagEstimator(ML, "text", "bag_of_words")
+                .Append(ML.Transforms.FeatureSelection.CountFeatureSelectingEstimator("bag_of_words", "bag_of_words_count", 10)
```

Comment:
>ML.Transforms.FeatureSelection [](start = 24, length = 30)

why you need full name?

Reply:
I wanted to test that I can access it from mlcontext

---
In reply to: [236791021](https://github.com/dotnet/machinelearning/pull/1683#discussion_r236791021) [](ancestors = 236791021)